buildSrc/src/main/java/Config.kt

```diff
@@ -80,6 +80,8 @@ object Config {
         private val retrofit2Group = "com.squareup.retrofit2"
         val retrofit2 = "$retrofit2Group:retrofit:$retrofit2Version"
         val retrofit2Gson = "$retrofit2Group:converter-gson:$retrofit2Version"
+
+        val coroutinesCore = "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.0"
```

Comment:
```suggestion
        val coroutinesCore = "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3"
```

1.5.0 depends on Kotlin 1.5.x, 1.4.3 depends on Kotlin 1.4.x, our projects right now are tested against Kotlin 1.4.30, so I'd stick to 1.4.3, wdyt?